behave_django/environment.py

```diff
@@ -21,8 +21,18 @@ def load_registered_fixtures(context):
     """
     Apply fixtures that are registered with the @fixtures decorator.
     """
+    # -- SELECT STEP REGISTRY:
+    # HINT: Newer behave versions use runner.step_registry
+    #   to be able to support multiple runners, each with its own step_registry.
```

Comment:
flake8 complaints out the length of this line. (I'm sorry!) :worried:

Reply:
Simply remove a single space in front of the line (before "to"), and the linting will pass and tests will start to run -- and hopefully pass!